errors building Exercise_8_Error_Handling

I built using these commands:
```
git clone https://github.com/codeplaysoftware/syclacademy.git
cd syclacademy/
git submodule update --init --recursive
mkdir build
cd build
CMAKE_PREFIX_PATH=/opt/hipSYCL/lib cmake ../ -DSYCL_ACADEMY_USE_HIPSYCL=ON -DHIPSYCL_PLATFORM=rocm -DHIPSYCL_GPU_ARCH=gfx906
make
```

Output [shortened]:
```
...
Scanning dependencies of target Exercise_8_source
[ 92%] Building SYCL object Code_Exercises/Exercise_8_Error_Handling/CMakeFiles/Exercise_8_source.dir/source.cpp.o
syclcc warning: No optimization flag was given, optimizations are disabled by default. Performance may be degraded. Compile with e.g. -O2/-O3 to enable optimizations.
/home/ak/my/dev/extern/syclacademy/Code_Exercises/Exercise_8_Error_Handling/source.cpp:15:1: error: main cannot be declared as global variable
int main {
^
/home/ak/my/dev/extern/syclacademy/Code_Exercises/Exercise_8_Error_Handling/source.cpp:16:31: error: expected '(' for function-style cast or type construction
   cl::sycl::default_selector device_selector;
   ~~~~~~~~~~~~~~~~~~~~~~~~~~ ^
/home/ak/my/dev/extern/syclacademy/Code_Exercises/Exercise_8_Error_Handling/source.cpp:28:2: error: expected ';' after top level declarator
}
 ^
 ;
3 errors generated when compiling for gfx900.
make[2]: *** [Code_Exercises/Exercise_8_Error_Handling/CMakeFiles/Exercise_8_source.dir/build.make:83: Code_Exercises/Exercise_8_Error_Handling/CMakeFiles/Exercise_8_source.dir/source.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:1245: Code_Exercises/Exercise_8_Error_Handling/CMakeFiles/Exercise_8_source.dir/all] Error 2
make: *** [Makefile:115: all] Error 2

```

--- a/Code_Exercises/Exercise_8_Error_Handling/source.cpp
+++ b/Code_Exercises/Exercise_8_Error_Handling/source.cpp
@@ -12,7 +12,7 @@
 
 class error_handling;
 
-int main {
+int main() {
    cl::sycl::default_selector device_selector;
 
    cl::sycl::queue queue(device_selector);

